sample/src/main/java/com/airbnb/airmapview/sample/MainActivity.java

```diff
@@ -93,10 +103,76 @@
 
   @Override public void onMapInitialized() {
     appendLog("Map onMapInitialized triggered");
-    map.animateCenterZoom(latLng, 10);
+    final LatLng airbnbLatLng = new LatLng(37.771883, -122.405224);
+    addMarker("Airbnb HQ", airbnbLatLng);
+    addMarker("Performance Bikes", new LatLng(37.773975,-122.40205));
+    addMarker("REI", new LatLng(37.772127, -122.404411));
+    map.animateCenterZoom(airbnbLatLng, 10);
+  }
+
+  private void addMarker(String title, LatLng latLng) {
     map.addMarker(new AirMapMarker(latLng, 1)
-                      .setTitle("Airbnb HQ")
-                      .setIconId(R.mipmap.icon_location_pin));
+                      .setTitle(title)
+                      .setIcon(getMarkerIcon(title)));
+  }
+
+  private Bitmap getMarkerIcon(String text) {
```

Comment:
should we perhaps include a sample icon in the resources instead of this drawing code?